### ISSUE TITLE
Coiled / Dask version dependency

### ISSUE THREAD

**Issue by rsignell-usgs** [coiled/feedback#252]
(this is Matt on Richard's computer)

```
$ coiled --version
Traceback (most recent call last):
  File "/home/rsignell/mambaforge/envs/pangeo/bin/coiled", line 6, in <module>
    from coiled.cli.core import cli
  File "/home/rsignell/mambaforge/envs/pangeo/lib/python3.9/site-packages/coiled/__init__.py", line 39, in <module>
    from .run import run
  File "/home/rsignell/mambaforge/envs/pangeo/lib/python3.9/site-packages/coiled/run.py", line 14, in <module>
    from .shutdown import NoClientShutdown
  File "/home/rsignell/mambaforge/envs/pangeo/lib/python3.9/site-packages/coiled/shutdown.py", line 6, in <module>
    from distributed.compatibility import PeriodicCallback  # pyright: reportPrivateImportUsage=false
ImportError: cannot import name 'PeriodicCallback' from 'distributed.compatibility' (/home/rsignell/mambaforge/envs/pangeo/lib/python3.9/site-packages/distributed/compatibility.py)
 rsignell@IGSAGI221LRSI10  ~
$ mamba list | grep coiled
coiled                    0.8.10             pyhd8ed1ab_0    conda-forge
 rsignell@IGSAGI221LRSI10  ~
$ mamba list | grep dask
dask                      2022.4.2           pyhd8ed1ab_0    conda-forge
dask-core                 2022.4.2           pyhd8ed1ab_0    conda-forge
dask-gateway              2022.4.0           pyh8af1aa0_0    conda-forge
dask-geopandas            0.2.0              pyhd8ed1ab_0    conda-forge
dask-image                2022.9.0           pyhd8ed1ab_0    conda-forge
qhub-dask                 0.4.4                hd8ed1ab_0    conda-forge
```

**Comment by rsignell-usgs** [comment]
Just a note that creating a fresh environment worked.

**Comment by jrbourbeau** [comment]
Thanks @rsignell-usgs! Glad to hear a fresh environment worked. FWIW the issue here had to do with compatibility with older versions of `distributed` (specifically `distributed<2022.10.1`). We just merged a fix into `coiled` that will fix this issue for `distributed<2022.10.1`
